index.d.ts

```diff
@@ -0,0 +1,86 @@
+export const version: number;
+export const validOptions: string[];
+
+export type Options = Record<string, unknown>;
```

Comment:
Please make the options correspond with this: https://developers.google.com/youtube/v3/docs/search/list